src/tree/taffy_tree.rs

```diff
@@ -221,7 +221,8 @@ impl<NodeContext> PrintTree for TaffyTree<NodeContext> {
 /// which makes the lifetimes of the context much more flexible.
 pub(crate) struct TaffyView<'t, NodeContext, MeasureFunction>
 where
-    MeasureFunction: FnMut(Size<Option<f32>>, Size<AvailableSpace>, NodeId, Option<&mut NodeContext>) -> Size<f32>,
+    MeasureFunction:
+        FnMut(Size<Option<f32>>, Size<AvailableSpace>, NodeId, Option<&mut NodeContext>, &Style) -> Size<f32>,
```

Comment:
As we need this quite often, what would you think about pulling it out in a type alias? Then we can guarantee it to be consistent and only have to update it in one place if further changes are necessary.

That would also give us a place to document the parameters in more detail, which I believe we are currently lacking.

Reply:
Yeah, I like a type alias here a lot.

Reply:
I think I tried a type alias, but it was breaking type inference for a simple closure. But I might try it again. I agree that it would help code clarity/maintainabiloty a lot.

Reply:
I'll look into this as a follow up.